docker/main/Dockerfile

```diff
@@ -8,7 +8,7 @@ ARG SLIM_BASE=debian:11-slim
 
 FROM ${BASE_IMAGE} AS base
 
-FROM --platform=${BUILDPLATFORM} debian:11 AS base_host
+FROM --platform=${BUILDPLATFORM} ${BASE_IMAGE} AS base_host
```

Comment:
Was this necessary? I think it was intentionally set to `debian:11` since jetson swaps in a different base image.

Reply:
I looked through the history when I changed this and I think it was just missed when `${BASE_IMAGE}` was introduced. Absolutely doesn't need to be a part of this PR.